### ISSUE TITLE
Possible --quiet mode

### ISSUE THREAD

**Issue by Nickiel12** [smasher164/pw-volume#31]
I was wondering if a `--quiet` flag, or something similar, could be added that suppresses the stderr output unless there was an error. 

I am using it for my `eww` sound control widget, and it makes it harder to see actual errors in the `eww logs` with all of the `pw-volume` output.

**Comment by smasher164** [comment]
Yeah this seems reasonable. We could add a `--quiet` flag for the `change` subcommand.
